test/helpers/setup-helper.js

```diff
@@ -89,10 +91,35 @@ window.document.body.appendChild(popoverContent);
 // eslint-disable-next-line no-shadow
 const { default: fetch, Headers, Request, Response } = require('node-fetch');
 
-Object.assign(window, { fetch, Headers, Request, Response });
+const shimmedFetch = (url, ...args) => {
+  return fetch(url, ...args).catch((error) => {
+    if (error.message === 'Only absolute URLs are supported') {
```

Comment:
For readability, should we move this block into a separate function like `handleRelativeFetchPaths`?

We could even import this shim from a separate file given the amount of logic involved?